### ISSUE TITLE
Is the app still maintained?

### ISSUE THREAD

**Issue by ghost** [maxrave-dev/SimpMusic#529]
Is the app still maintained?

**Comment by CrazyWolf13** [comment]
Yes of course?!<br>Just take a look at how many releases were published in the last time.[https://github.com/maxrave-dev/SimpMusic/releases](https://github.com/maxrave-dev/SimpMusic/releases?)
